tests/unit/source_code/samples/functional/parent_that_magic_runs_child_that_uses_value_from_parent.py

```diff
@@ -0,0 +1,12 @@
+# Databricks notebook source
+
+some_table_name = "some_table_name"
```

Comment:
```suggestion
some_table_name = "old.things"
```
use one of the tables from mock migration index to trigger the `table-is-migrated` problem

Reply:
we're already triggering the `default-format-changed-in-dbr8` problem, why do you want to trigger 2 problems ?

Reply:
done